translating-guide/en/translate-documentation.md

```diff
@@ -117,17 +117,25 @@ You may find that the machine translator will skip some sections, which you will
 
 You will have to click **Submit** below each piece of text to confirm that the translation is complete. 
 
-### Glossary
+#### Glossary
 
 PKP software translators maintain glossaries on a per-language basis of standard translations of technical terms. You should try to use the same translations when possible. You can check the glossaries in the [software translation tool](https://translate.pkp.sfu.ca/dictionaries/pkp-lib/). 
 
 You can also check a translated installation of OJS, OMP, or OPS to see what terms are used in the application itself. 
 
-### Screenshots
+#### Screenshots
 
-You will see screenshots as you are translating, but you can ignore them. PKP has an automatic process to generate screenshots for translations.
+You will see screenshots as you are translating, but you can ignore them. PKP is developing an automatic process to generate screenshots for translations.
 
-## Create Review Request
+<!--- You will see screenshots as you are translating. PKP is developing an automatic process to generate screenshots for translations. Until that is available, you are welcome to take screenshots of the software that will be added to the guide. Otherwise, English screenshots will be used until the other language screenshots are available.
```

Comment:
Is this intentionally commented? Should I review it?

Reply:
I commented it out because I still need to test how this process works.  This is only my guess.  I will publish it properly after I test it.